docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -44,7 +44,30 @@ Before enabling the Telemetry feature, a few setup steps are required to ensure
 
```

Comment:
In point 2 above can we link to AWS Distro for OpenTelemetry Collector's download page?